docs/devel/scheduler_algorithm.md

```diff
@@ -40,7 +40,7 @@ For each unscheduled Pod, the Kubernetes scheduler tries to find a node across t
 
 The purpose of filtering the nodes is to filter out the nodes that do not meet certain requirements of the Pod. For example, if the free resource on a node (measured by the capacity minus the sum of the resource requests of all the Pods that already run on the node) is less than the Pod's required resource, the node should not be considered in the ranking phase so it is filtered out. Currently, there are several "predicates" implementing different filtering policies, including:
 
-- `NoDiskConflict`: Evaluate if a pod can fit due to the volumes it requests, and those that are already mounted.
+- `NoDiskConflict`: Evaluate if a pod can fit due to the volumes it requests, and those that are already mounted. Currently supported volumes are: AWS EBS, GCE PD, and Ceph RBD. Note, Persistent Volumes are not evaluated, and thus pods that use Persistent Volume Claims are not constrained by this policy.
```

Comment:
Then shouldn't this be:

`Only Persistent Volume Claims for those supported types are checked.  Persistent Volumes added directly to pods are not evaluated and are not constrained by this policy.`